index.html

```diff
@@ -1,185 +1,198 @@
+    </article>
+    <article class="featured flex white-bg">
+      <h2>Professional Art<br>Printing Data</h2>
+      <p>
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required. has been the industry's standard
+      </p>
+      <ul class="tech flex">
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="submit">See Project</button>          
+    </article>   
+  </section>
+  <article class="about-me theme-color">
+    <h1>About me</h1>
+      <p>
+        Hello I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don’t hestiate to
+        contact me.
+      </p>
+      <button type="submit">Get My Resume</button>
+      <hr>
+  </article>
+  <section class="lfs theme-color">
+    <ul class="outer">
+      <li class="outer"><img src="imgs/fmliststyle.png" alt="list-style-img">  Language
+        <ul class="inner">
+          <li class="inner">Javascript</li>
+          <li class="inner">Ruby</li>
+          <li class="inner">Html</li>
+          <li class="inner">Css</li>
+        </ul>
+      </li> 
+      <li class="outer"><img src="imgs/fwliststyle.png" alt="list-style-img">  Frameworks
+        <ul class="inner">
+          <li class="inner">Bootstrap</li>
+          <li class="inner">Ruby on Rails</li>
+          <li class="inner">Rspec</li>
+          <li class="inner">CapyBara</li>
+          <li class="inner">Selenium</li>
+        </ul>
+      </li>
+      <li class="outer"><img src="imgs/skliststyle.png" alt="list-style-img">  Skills
+        <ul class="inner">
+          <li class="inner">Codekit</li>
+          <li class="inner">Github</li>
+          <li class="inner">Codepen</li>
+          <li class="inner">Gitlab</li>
+          <li class="inner">Terminal</li>
+        </ul>
+      </li>
+    </ul>
+  </section>
+  <section class="contact white-bg">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/xvolveaz" method="post">
+      <ul>
         <li>
-          <a href="https://www.linkedin.com/in/faruq-hammed-931862169/"
-            ><img src="imgs/linkedin.png" alt="linkedin-icon"
-          /></a>
+          <input type="text" name="full_name" maxlength="30" placeholder="Full name" required>
         </li>
         <li>
-          <a href="#"><img src="imgs/angelist.png" alt="angelist-icon" /></a>
+          <input type="email" name="email" placeholder="Email address" required>
         </li>
         <li>
-          <a href="https://twitter.com/mansa_susa"
-            ><img src="imgs/twitter.png" alt="twitter-icon"
-          /></a>
+          <textarea name="message" id="message" maxlength="500" required>Enter text here</textarea>
```

Comment:
### OPTIONAL

- [ ] Kindly add "Enter text here" as a placeholder so users don't need to clear text before typing their message in the textarea👍🏿

Reply:
> ### OPTIONAL
> * [ ]  Kindly add "Enter text here" as a placeholder so users don't need to clear text before typing their message in the textarea👍🏿

Ok I'll do that, thank you for the recommendation.